Dockerfile

```
@@ -0,0 +1,28 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION} AS base
+
+
+WORKDIR /devops_todolist_docker
+
+
+COPY requirements.txt requirements.txt
+
+COPY . .
```

The step `COPY requirements.txt requirements.txt` is redundant because `COPY . .` copies all files from the local folder into the container."
```suggestion
COPY . .
```